packages/client/src/utils/str.ts

```diff
@@ -0,0 +1,8 @@
+const hyphenateRE = /\B([A-Z])/g;
+export function hyphenate(str: string): string {
+  return str.replace(hyphenateRE, "-$1").toLowerCase();
+}
+
+export function snakelize(str: string): string {
+  return hyphenate(str).replace(/-/g, "_");
```

Comment:
honestly I would just use lodash for this

Reply:
I took what was done previously in SM.